package.json

```diff
@@ -25,13 +25,11 @@
   "homepage": "https://github.com/PoonLab/covizu#readme",
   "dependencies": {
     "compression": "^1.7.4",
+    "cypress": "^12.5.1",
     "dotenv": "^16.0.1",
     "express": "^4.17.2",
     "fs": "^0.0.1-security",
     "http": "^0.0.1-security",
     "https": "^1.0.0"
-  },
-  "devDependencies": {
-    "cypress": "^12.1.0"
   }
```

Comment:
* Does `cypress` need to be added as a dependency?
* Do we need to use version `12.5.1`